### DISCUSSION TITLE
hy不支持cdn，有没有方案可以隐藏服务器真实IP？

### DISCUSSION THREAD

**Discussion by kilvn** [apernet/hysteria#439] [Q&A] [answered]
一般个人的站用CDN可以隐藏源IP，相对安全一点，但是hy不支持CDN，cloudflare试过了，那有没有什么方案可以做到类似CDN的隐藏服务器真实IP呢？

**Comment by cross-hello** [comment, marked as answer]
Why not try cloud flared tunnel. In the same way could proxy your TCP traffic between your server and cdn server.

Sep 17, 2022 18:32:19 东子 ***@***.***>:

**Reply by zc08** [reply]
Hi, I'm trying to use cloudflared tunnel with hysteria but have been unsuccessful, would it be possible to share a cloudflared config.yaml file? Thank you so much!
